lib/spack/spack/fetch_strategy.py

```diff
@@ -940,6 +940,126 @@ def __str__(self):
         return '[git] {0}'.format(self._repo_info())
 
 
+@fetcher
+class CvsFetchStrategy(VCSFetchStrategy):
+
+    """Fetch strategy that gets source code from a CVS repository.
+       Use like this in a package:
+
+           version('name',
+                   cvs=':pserver:anonymous@www.example.com:/cvsroot%module=modulename')
+
+       Optionally, you can provide a branch and/or a date for the URL:
+
+           version('name',
+                   cvs=':pserver:anonymous@www.example.com:/cvsroot%module=modulename',
+                   branch='branchname', date='date')
```

Comment:
Does CVS have branches? You didn't mention this feature in the docs

Reply:
It does have branches. I updated the documentation.